apps/web/services/feed/index.ts

```diff
@@ -52,9 +55,11 @@ const fetch = memoize(
           },
         },
       },
-      orderBy: [{ lastActivityAt: 'desc' }, { lastReplyAt: 'desc' }],
-      take,
-      skip,
+      orderBy: [
+        // { lastActivityAt: 'desc' },
```

Comment:
this was for @cheeseblubber, do we want to keep this behaviour?

Reply:
We probably can use the lastActivityAt